src/chart/graph/simpleLayoutHelper.js

```diff
@@ -31,12 +34,16 @@ export function simpleLayout(seriesModel) {
         node.setLayout([+model.get('x'), +model.get('y')]);
     });
 
-    simpleLayoutEdge(graph);
+    simpleLayoutEdge(graph, seriesModel);
 }
 
-export function simpleLayoutEdge(graph) {
-    graph.eachEdge(function (edge) {
-        var curveness = edge.getModel().get('lineStyle.curveness') || 0;
+export function simpleLayoutEdge(graph, seriesModel) {
+    graph.eachEdge(function (edge, index) {
+        var curveness = zrUtil.retrieve3(
+            edge.getModel().get('lineStyle.curveness') || null,
```

Comment:
Is there necessay to add `|| null`?
If failed to get `lineStyle.curveness` from model, it will return `undefined`, and as we known, `undefined == null`.

Reply:
because the zrUtil.retrieve3 use ```!=```, so i think ```|| null``` is necessay?
```export function retrieve3(value0, value1, value2) {
    return value0 != null
        ? value0
        : value1 != null
        ? value1
        : value2;
}```

Reply:
Just `!=` but not `!==`.
`undefined != null` is `false`, so I think there is no need to worry about it.

Reply:
oh, yes, i remove it.